controllers/src/test/scala/cromwell/pipeline/controller/AkkaHttpClientTest.scala

```diff
@@ -0,0 +1,158 @@
+package cromwell.pipeline.controller
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.model.StatusCodes
+import akka.stream.ActorMaterializer
+import com.github.tomakehurst.wiremock.WireMockServer
+import com.github.tomakehurst.wiremock.client.WireMock._
+import com.github.tomakehurst.wiremock.core.WireMockConfiguration._
+import org.scalatest.{ AsyncWordSpec, BeforeAndAfterEach, Matchers }
+import org.scalatestplus.mockito.MockitoSugar
+
+import scala.concurrent.ExecutionContext
+
+class AkkaHttpClientTest extends AsyncWordSpec with Matchers with MockitoSugar with BeforeAndAfterEach {
+  implicit val actorSystem: ActorSystem = ActorSystem()
+  implicit val materializer: ActorMaterializer = ActorMaterializer()
+  implicit val ec: ExecutionContext = actorSystem.dispatcher
+
+  val wireMockServer = new WireMockServer(wireMockConfig().dynamicPort())
+
+  override def beforeEach: Unit =
+    wireMockServer.start()
+
+  override def afterEach: Unit =
+    wireMockServer.stop()
```

Comment:
When using methods like `beforeAll`, `beforeEach`, `afterAll`, `afterEach` a good practice is to call corresponding methods from parent. I.e.
```
  override def beforeEach: Unit = {
    super.beforeEach
    wireMockServer.start()
}
```
and
```
override def afterEach: Unit = {
    wireMockServer.stop()
    super.afterEach
}
```
You never know what these methods do. Omitting them may cause some bugs.

Reply:
Thank you for your advice, changed it